app/assets/javascripts/oxalis/model/reducers/volumetracing_reducer.js

```diff
@@ -81,14 +81,6 @@ function VolumeTracingReducer(state: OxalisState, action: VolumeTracingAction):
           return addToLayerReducer(state, volumeTracing, action.position);
         }
 
-        case "FINISH_EDITING": {
-          if (volumeTracing.contourList.length > 0) {
-            return addToLayerReducer(state, volumeTracing, volumeTracing.contourList[0]);
```

Comment:
This is unnecessary because the volumelayer will connect the first and last point in the contour anyways (in trace mode).